libbpfgo.go

```diff
@@ -611,6 +611,14 @@ func (b *BPFMap) ValueSize() int {
 	return int(C.bpf_map__value_size(b.bpfMap))
 }
 
+func (b *BPFMap) SetValueSize(size uint32) error {
+	errC := C.bpf_map__set_value_size(b.bpfMap, C.uint(size))
```

Comment:
Perhaps we could read both the return value and the error, as reading just one of the return values will result in us reading the C function return value, but not the errno

Reply:
bpf_map__set_value_size() returns the error directly and does not set errno.